### DISCUSSION TITLE
Anyone would be willing to help port this mod to the latest experimental and keep it up to date on a monthly basis?

### DISCUSSION THREAD

**Discussion by KestasV** [FrontierMods/Armory#2] [Ideas]
I would love to support this, well. Up to you folks to decide!

**Comment by Hyperseeker** [comment]
With the amount of man-hours that's already gone into this (as of yet unreleased) mod, I feel confident in saying I won't be able to support two versions of it at the same time.

If anyone *does* have interest in doing this, you have my blessing. (You don't need my permission, since the mod is licensed under a very permissive MIT license.)
